Documentation/contributor-guide/roadmap.md

```diff
@@ -0,0 +1,54 @@
+# roadmap
```

Comment:
Is it worth including grpc-proxy? 
- #15480

Reply:
It is isn't blocking anything, not even been discussed.

There are two related tool: `grpc-proxy `and `gateway`, I think we should handle them in the same way. Let's evaluate it separately. I suggest not to get them included in the roadmap for now. As we discussed above, the roadmap isn't set in stone, it may shift over time.